src/mapeo-project.js

```diff
@@ -102,7 +104,8 @@ export class MapeoProject {
     const sqlite = new Database(dbPath)
     const db = drizzle(sqlite)
     migrate(db, {
-      migrationsFolder: new URL('../drizzle/project', import.meta.url).pathname,
+      migrationsFolder: new URL(projectMigrationFolder, import.meta.url)
```

Comment:
```suggestion
      migrationsFolder: projectMigrationsFolder,
```

Reply:
do you think the URL should be build before passing to to MapeoManager and MapeoProject (so, in charge of the consumer?) ?

Reply:
yep! would say it provides the most flexibility and correctness based on the context of the consumer (e.g. helps to have directory flexibility in the mobile app due to app structure)